cmd/wallet-sdk-js/package.json

```diff
@@ -8,7 +8,7 @@
     "build:assets": "bash scripts/build_assets.sh",
     "build": "rm -rf dist/* && npm run build:assets && npx webpack --config ./webpack.config.js --mode=production",
     "prepare": "npm run build",
-    "test": "karma start --single-run"
+    "test": "karma start --single-run -- --foo"
```

Comment:
What do we need the `-- --foo` arguments for?

Reply:
removed